parse.go

```diff
@@ -67,6 +66,19 @@ func parseIter(set *flag.FlagSet, ip iterativeParser, args []string, shellComple
 	}
 }
 
+const providedButNotDefinedErrMsg = "flag provided but not defined: -"
```

Comment:
How would this work with flags having two hyphens as a prefix?

Reply:
I tested it locally and it works the same. The error message is always this constant.